### ISSUE TITLE
NbSelect touched property not working in Reactive Forms

### ISSUE THREAD

**Issue by KonstantinosD85** [akveo/nebular#1629]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**
While using the NbSelectComponent inside a Reactive Form the touched property is set to true when i click an other Control or even some random place in the page

**Expected behavior:**
The touched property of the a control bound to a NbSelectComponent should be set to true ONLY if the specific control is touched.

**Steps to reproduce:**
In the provided StackBlitz code click the Input field or even a random point and watch the touched property of the nbSelect field (it is bound to the template). 

**Related code:**

[StackBlitz Sample Code](https://stackblitz.com/edit/github-3mabhk)

```
insert short code snippets here
```

### Other information:

**npm, node, OS, Browser**
```
 `node --version` : v11.13.0
 `npm --version`: 6.7.0
OS: Linux (Fedora 30)
Browser: Chrome/Firefox
```

**Angular, Nebular**
```
    "@angular/animations": "~8.0.0",
    "@angular/cdk": "~8.0.0",
    "@angular/common": "~8.0.0",
    "@angular/compiler": "~8.0.0",
    "@angular/core": "~8.0.0",
    "@angular/fire": "^5.2.1",
    "@angular/forms": "~8.0.0",
    "@angular/platform-browser": "~8.0.0",
    "@angular/platform-browser-dynamic": "~8.0.0",
    "@angular/router": "~8.0.0",
    "@nebular/theme": "^4.0.0",

```

**Comment by BranislavCulak** [comment]
same problem here

**Comment by nilpaco** [comment]
I have the same issue using nbSelect

**Comment by yggg** [comment]
Fixed in #1878

**Comment by maihannijat** [comment]
@yggg It does seem fixed. Nebular 5 and Angular 8

The `NbSelectComponent` in a ReactiveForm is marked as touched with no interaction with the form.
